test/cookbooks/app_test/recipes/app2.rb

```diff
@@ -1,36 +1,32 @@
-include_recipe 'osl-mysql::server'
+node.force_override['osl-mysql']['enable_percona_client'] = false
```

Comment:
Why is this still needed with the `osl_mysql_test` resource?

Reply:
I forgot that I have removed this attribute from the `kitchen.yml`, so this is no longer required. It's been removed.